types/setting.go

```diff
@@ -757,6 +761,16 @@ var (
 		ReadOnly: false,
 		Default:  "false",
 	}
+
+	SettingDefinitionOrphanAutoDeletion = SettingDefinition{
+		DisplayName: "Orphan Auto-Deletion",
+		Description: "This setting allows Longhorn to delete the orphaned data and its corresponding resource automatically.",
```

Comment:
NIT: mentioning that the Orphans on the down/unknown nodes won't be cleaned up automatically

Reply:
Precisely, the orphan CR's `DataCleanable` condition will be set to be `false` in the orphan controller because the node is down, but the orphaned data won't be deleted.

Reply:
I knew. 
It's OrphanController's duty to check the condition and delete the actual data. 
For this setting, it's NodeController that relies on it to determine if a deletion timestamp can be added to the CRs.

Reply:
Got your point. Thank you.